### ISSUE TITLE
Q: How to find out which audio is playing

### ISSUE THREAD

**Issue by Alanko5** [kingslay/KSPlayer#85]
@kingslay please help me.
How to find out which audio is playing? 

get all audio tracks:
`player.tracks(mediaType: .audio)`

How to find out which subtitles are set?
'srtControl.filterInfos'

how to get the information that is selected?

**Comment by Alanko5** [comment]
This is right way?
```
extension CustomVideoPlayerView {
    public func getSubtitlesTrack() -> [KSPlayer.SubtitleInfo] {
        var subtitles:[KSPlayer.SubtitleInfo] = [URLSubtitleInfo(subtitleID: "", name: "disabled".localizable)]
        subtitles.append(contentsOf: self.srtControl.filterInfos {  _ in return true })
        return subtitles
    }
    
    public func getSelectedSubtitle() -> KSPlayer.SubtitleInfo? {
        if let subtitleView = self.srtControl.view as? KSSubtitleView {
            return subtitleView.selectedInfo.wrappedValue
        }
        return nil
    }
    
    public func setSubtitle(_ new: KSPlayer.SubtitleInfo) {
        if let subtitleView = self.srtControl.view as? KSSubtitleView {
            subtitleView.selectedInfo.wrappedValue = new
        }
    }
}
```

**Comment by ridhabnj** [comment]
Hi,

Thanks, but not working.

How to change this :         

`guard let player = self.playerLayer.player else { return [] }
`

i have :     
`let LivePlayer = CustomVideoPlayerView()
`

Can you help ?

**Comment by kingslay** [comment]
i fixed 88ca75bc6e0c33aebf1de60c7902950dc1c551de  

```
public protocol MediaPlayerTrack {
    var name: String { get }
    var language: String? { get }
    var mediaType: AVFoundation.AVMediaType { get }
    var fps: Int { get }
    var rotation: Double { get }
    var bitRate: Int64 { get }
    var naturalSize: CGSize { get }
    var isEnabled: Bool { get }
}
```

**Comment by Alanko5** [comment]
Thx
